long_term/test_load_long.py

```diff
@@ -0,0 +1,76 @@
+# pylint: skip-file
+
+"""Test file for load functions."""
+
+from unittest.mock import patch, MagicMock
```

Comment:
Add line between native modules, external modules and functions imported from local files.